pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: intl_utils
 description: intl_utils is a dart library that generates Dart localization code from ARB file. Generated code relies on Intl library.
-version: 1.9.0
+version: 1.9.1
```

Comment:
Null safety version should be `2.0.0-nullsafety`
ref https://dart.dev/null-safety/migration-guide#package-version

Reply:
This PR neither lets the tool produce null-safe code nor is the tool's code null-safe. So the linked guide does definitely not apply.

To be precise, the `-nullsafety` part marks a version as a pre-release (per semantic versioning). This is done because Dart 2.12 (which is not yet released) is required.\
This PR ist working on Dart 2.2.2 onwards (if the pubspec sdk constraint was right previously)

Reply:
Understand, thanks~